tests/expected/oc/50631-nl_oc_msg_args_min_params.m

```diff
@@ -0,0 +1,8 @@
+static void function() {
+	[object param1:nil
+	 param2:nil
+	 param3:nil
+	 param4:nil];
+
+	[object param1:nil param2:nil param3:nil];
```

Comment:
I don't use OC, so maybe I am saying something wrong. Based on the description of the new option, I would expect functions/objects/[whatever is the right term in OC] to be broken up in multiple lines when they have 3 (in this specific test) or more parameters. So why is this second case not broken up? I would expect this
```
[object param1:nil param2:nil];
```
not to break up, while the two examples in the test file to be broken up.
Am I missing something?

Reply:
Good catch :) We're off by one here because param1 counts as a function name chunk. I've updated the code to instead look for colons to count parameters. Updated the tests as well.